__mockData__/67136336f73537cfd8a1fede6db932bd94d20423.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Attributes": {
+    "psid": "1",
+    "morning": 0,
+    "evening": 0
+  }
+}
```

Comment:
Consider putting mock data for requests and dynamodb queries in seperate directories

Reply:
:ok_hand: